docs/distributed/README.md

```diff
@@ -66,7 +69,13 @@ export MINIO_SECRET_KEY=<SECRET_KEY>
 minio server http://host{1...32}/export{1...32} http://host{33...64}/export{1...32}
 ```
 
-Now the server has expanded storage of *1024* more disks in total of *2048* disks, new object upload requests automatically start using the least used cluster. This expansion strategy works endlessly, so you can perpetually expand your clusters as needed.  When you restart, it is immediate and non-disruptive to the applications. Each group of servers in the command-line is called a zone. There are 2 zones in this example. Objects are placed in zones based on which zone has the *fmost free* space. Within each zone, the location of the erasure-set of drives is determined based on a deterministic hashing algorithm.
+Now the server has expanded storage of *1024* more disks in total of *2048* disks, new object upload requests automatically start using the least used cluster. This expansion strategy works endlessly, so you can perpetually expand your clusters as needed.  When you restart, it is immediate and non-disruptive to the applications. Each group of servers in the command-line is called a zone. There are 2 zones in this example. New objects are placed in zones in proportion to the amount of free space in each zone. Within each zone, the location of the erasure-set of drives is determined based on a deterministic hashing algorithm.
+
+*Note*
+
+Each zone you add must have the same erasure coding set size as the original zone, so the same data redundancy SLA is maintained.
+
+For example, if your first zone was 8 drives, you could add further zones of 8 drives each, but not a zone of 16 drives.  That's because 16 drives are treated as a single EC set of 16, not two sets of 8.
```

Comment:
FWIW it is allowed to pick the set count so where this actually doesn't need to be true.

```
export MINIO_ERASURE_SET_DRIVE_COUNT=8
```

So we have to mention that original SLA is what is going to be always honored, so it better to deploy an initial cluster with large enough SLA to maintain this SLA in future.

Reply:
We can actually do it ourselves too, such that secondary zones automatically can inherit the SLAs of first zone. This would allow users to expand atleast in divisible units..

Reply:
Doesn't seem to work for me:

```
root@minio1:~# ./minio server http://minio1/ec/{1...4}
Attempting encryption of all config, IAM users and policies on MinIO backend
<< wait a few mins >>
<< ok >>
^C

root@minio1:~# MINIO_ERASURE_SET_DRIVE_COUNT=4 ./minio server http://minio1/ec/{1...4} http://minio1/ec/{5...12}
ERROR Invalid command line arguments: All zones should have same drive per set ratio - expected 4, got 8
root@minio1:~# export MINIO_ERASURE_SET_DRIVE_COUNT=4
root@minio1:~# ./minio server http://minio1/ec/{1...4} http://minio1/ec/{5...12}
ERROR Invalid command line arguments: All zones should have same drive per set ratio - expected 4, got 8
```

This is with `minio version 2019-12-30T05:45:39Z`

Reply:
Retested with `minio version RELEASE.2020-01-03T19-12-21Z` - same result

Reply:
Ah okay, this assumption needs a fix. Perhaps this wording can be removed later when we fix it.